steps/cli/oc.go

```diff
@@ -187,5 +190,17 @@ var _ = gauge.Step("Configure the bundles resolver", func() {
 })
 
 var _ = gauge.Step("Enable console plugin", func() {
+	openshiftVersion := openshift.GetOpenShiftVersion(store.Clients())
+	minorVersion, err := strconv.Atoi(strings.Split(openshiftVersion, ".")[1])
```

Comment:
if the `GetOpenShiftVersion` returns `""`, the `strings.Split(openshiftVersion, ".")[1]` will panic.

Reply:
fixed